src/server/rpc/procedures/utils/api-consumer.js

```diff
@@ -80,6 +85,15 @@ class ApiConsumer {
         });
     }
 
+    _getCacheKey(queryOptions){
+        let parameters = [];
+        parameters.push(queryOptions.method || 'GET');
+        let fullUrl = (queryOptions.baseUrl || this._baseUrl) + queryOptions.queryString;
+        parameters.push(fullUrl);
+        if (queryOptions.body) parameters.push(queryOptions.body);
+        return parameters.join('&');
```

Comment:
Pretty minor thing but I would not join them with '&' as it is standard in http requests and could cause issues. I would join it with a character that is not used in urls.

I mean that there is ambiguity with this joining scheme: `POST&https://some.api?additional=thing&something=else&isThis=theBody` is not clear on what part is the body. To be fair, POST requests don't really use querystring stuff so it should be fine but it seems unnecessary to even have this risk when you could just use something like `#` and you should avoid this all together (bc hash values are not sent to the server by the browser).